runtime/src/bank.rs

```diff
@@ -4642,7 +4649,8 @@ impl Bank {
                         .unwrap_or_default()
                 });
 
-            ((prioritization_fee
+            ((prioritization_fee_rate
+                .saturating_mul(PRIORITIZATION_FEE_RATE_UNITS)
```

Comment:
The prioritization fee is calculated differently based on whether the `prioritization_fee_type_change` feature is activated or not. If it's activated we can multiply by the tick size here, otherwise we should keep it as it was. It would be nice if you calculated the prioritization fee on line 4627 and handled the feature branch there so that this line can remain as it was (`prioritization_fee`)

Reply:
god catch .. thanks! fixed it